logstash.go

```diff
@@ -72,20 +111,25 @@ func (a *LogstashAdapter) Stream(logstream chan *router.Message) {
 		}
 
 		tags := GetContainerTags(m.Container, a)
+		fields := GetLogstashFields(m.Container, a)
 
 		var js []byte
 		var data map[string]interface{}
 
 		// Parse JSON-encoded m.Data
 		if err := json.Unmarshal([]byte(m.Data), &data); err != nil {
-			// The message is not in JSON, make a new JSON message.
-			msg := LogstashMessage{
-				Message: m.Data,
-				Docker:  dockerInfo,
-				Stream:  m.Source,
-				Tags:    tags,
-			}
-
+			
+		    msg := make(map[string]interface{}) 
+
+			msg["message"] = m.Data
+			msg["docker"] = dockerInfo
+		    msg["stream"] = m.Source
+		    msg["tags"] = tags
+
+			for k, v := range fields { 
```

Comment:
It could be good to blacklist certain keys as to not accidentally overwrite docker, stream, tags etc. What do you think?